pkg/commands/cmdlib/bundleio.go

```diff
@@ -202,3 +204,40 @@ func formatFromFile(path string) string {
 	}
 	return ""
 }
+
+// ParseAnnotations parses CLI flag format for annotations
+func ParseAnnotations(p string) map[string]string {
```

Comment:
nit: really something like ParseKeyValueFlag

Reply:
I left it for now, but maybe if you change to the "=" format. By the way, in patch applier you have a comment about whether a whole filter is needed or just annotations. I think just annotations may be cleaner. What happens if they have a filter with "kind"? it have unexpected effects.

I followed the example and did a full filter, but we may want to change that.